Makefile

```diff
@@ -6,6 +6,7 @@ install-python:
 	pip install "setuptools>=0.9.8"
 	# order matters here, base package must install first
 	pip install -e .
+	pip install ujson
```

Comment:
This is being done manually to force Travis to install and run tests against it.